src/components/Header/StatusBadge.tsx

```diff
@@ -82,60 +76,70 @@ const StagingPopoverContent = ({ status }: StagingPopoverContentProps) => {
   )
 }
 
-export const StatusBadge = (): JSX.Element => {
-  const { siteName } = useParams<{ siteName: string }>()
-  const {
-    data: getStagingStatusData,
-    isLoading: isGetStagingStatusLoading,
-  } = useGetStagingStatus(siteName)
-  const { status } = getStagingStatusData || {}
-  if (!status || isGetStagingStatusLoading) {
-    return <> </>
-  }
+export const StatusBadgeComponent = (
+  status: BuildStatus | undefined
+): JSX.Element => {
   let displayText = ""
   let colourScheme = ""
-  let dotColor = "#505660"
+
   switch (status) {
     case "PENDING":
       displayText = "Updating staging site"
       colourScheme = "grey"
-      dotColor = "#505660"
+
       break
     case "READY":
       displayText = "Staging site is ready"
       colourScheme = "success"
-      dotColor = "#0F796F"
+
       break
     case "ERROR":
       displayText = "Staging site is out of date"
       colourScheme = "warning"
-      dotColor = "#FFDA68"
+
       break
     default:
       break
   }
 
+  if (!status) {
+    return <> </>
+  }
   return (
-    <Box position="relative">
-      <Popover trigger="hover" placement="bottom">
-        <PopoverTrigger>
+    <Popover placement="bottom-start" trigger="hover">
+      <PopoverTrigger>
+        <Box mr="1rem">
           <Badge
             colorScheme={colourScheme}
             variant="subtle"
             cursor="default"
             borderRadius="3.125rem"
           >
-            <GoDotFill size="1rem" color={dotColor} />
+            <Icon as={GoDotFill} size="1rem" />
+
             <Text ml="0.25rem" mr="0.5rem">
               {displayText}
             </Text>
-            <BsFillQuestionCircleFill color="#454953" />
           </Badge>
-        </PopoverTrigger>
-        <PopoverContent width="26.25rem" height="auto" mt="1.75rem">
+        </Box>
+      </PopoverTrigger>
+
+      <PopoverContent width="26.25rem" height="auto">
+        <PopoverBody>
           <StagingPopoverContent status={status} />
-        </PopoverContent>
-      </Popover>
-    </Box>
+        </PopoverBody>
+      </PopoverContent>
+    </Popover>
   )
 }
+export const StatusBadge = (): JSX.Element => {
+  const { siteName } = useParams<{ siteName: string }>()
+  const { data: getStagingStatusData } = useGetStagingStatus(siteName)
+  const status = getStagingStatusData?.status
+
+  const badge = useMemo(() => {
+    return StatusBadgeComponent(status)
+  }, [status])
+
+  return badge
```

Comment:
not important to fix, but this `useMemo` is redundant - the `StatusBadgeComponent` will re-render on props change, which consists only of `status`. similarly, the `useMemo` will recompute on deps change, which is also `status`. 

this is identical to 

```ts
return <StatusBadgeComponent status = {status} />
```